benchmark/benchmark_view_access.cpp

```diff
@@ -241,28 +236,29 @@ namespace xt
         auto calc_dofval(xt::xtensor<double, 3, L>& elemvec, xt::xtensor<double, 1, L>& dofval)
         {
             dofval.fill(0.0);
-            for (size_t e = 0 ; e < M_NELEM ; ++e)
-                for (size_t m = 0 ; m < M_NNE ; ++m)
-                    for (size_t i = 0 ; i < M_NDIM; ++i)
+            for (size_t e = 0; e < M_NELEM; ++e)
+                for (size_t m = 0; m < M_NNE; ++m)
+                    for (size_t i = 0; i < M_NDIM; ++i)
                         dofval(m_dofs(m_conn(e, m), i)) += elemvec(e, m, i);
         }
 
         auto calc_dofval_simple(simple_array<double, 3>& elemvec, simple_array<double, 1>& dofval)
         {
             dofval.fill(0.0);
-            for (size_t e = 0 ; e < M_NELEM ; ++e)
-                for (size_t m = 0 ; m < M_NNE ; ++m)
-                    for (size_t i = 0 ; i < M_NDIM; ++i)
+            for (size_t e = 0; e < M_NELEM; ++e)
+                for (size_t m = 0; m < M_NNE; ++m)
+                    for (size_t i = 0; i < M_NDIM; ++i)
                         dofval(m_dofs(m_conn(e, m), i)) += elemvec(e, m, i);
         }
 
         auto calc_unchecked_dofval(xt::xtensor<double, 3, L>& elemvec, xt::xtensor<double, 1, L>& dofval)
         {
             dofval.fill(0.0);
-            for (size_t e = 0 ; e < M_NELEM ; ++e)
-                for (size_t m = 0 ; m < M_NNE ; ++m)
-                    for (size_t i = 0 ; i < M_NDIM; ++i)
-                        dofval.unchecked(m_dofs.unchecked(m_conn.unchecked(e, m), i)) += elemvec.unchecked(e, m, i);
+            for (size_t e = 0; e < M_NELEM; ++e)
+                for (size_t m = 0; m < M_NNE; ++m)
+                    for (size_t i = 0; i < M_NDIM; ++i)
+                        dofval.unchecked(m_dofs.unchecked(m_conn.unchecked(e, m), i)
+                        ) += elemvec.unchecked(e, m, i);
```

Comment:
We should format this by hand, with a temporary

Reply:
Either that, `// clang-format: off`, or using trailing comments to force line splits.

There are a couple of failures like this from `clang-format`. May I suggest we leave then as-is for now and change them when we work on relevant piece of codes?

Reply:
I think that we should change the code, because it was just poorly written (by me ;)). But I'm fine to do it in a separate PR after. We should just keep a list

Reply:
https://github.com/xtensor-stack/xtensor/issues/2632